features/fields-mapping.csv

```diff
@@ -0,0 +1,667 @@
+kinderen.naam.inOnderzoek.geslachtsnaam;kinderen.naam.inOnderzoek.geslachtsnaam
+kinderen.naam.inOnderzoek.voorletters;kinderen.naam.inOnderzoek.voorletters
+kinderen.naam.inOnderzoek.voornamen;kinderen.naam.inOnderzoek.voornamen
+kinderen.naam.inOnderzoek.voorvoegsel;kinderen.naam.inOnderzoek.voorvoegsel
+kinderen.naam.voorletters;kinderen.naam.voorletters
+kinderen.naam.voornamen;kinderen.naam.voornamen
+kinderen.naam.voorvoegsel;kinderen.naam.voorvoegsel
+kinderen.indicatieOnbekend;kinderen.indicatieOnbekend
+leeftijd;leeftijd
+naam;naam
+naam.adellijkeTitelPredicaat;naam.adellijkeTitelPredicaat
+naam.adellijkeTitelPredicaat.code;naam.adellijkeTitelPredicaat.code
+naam.adellijkeTitelPredicaat.omschrijving;naam.adellijkeTitelPredicaat.omschrijving
+naam.adellijkeTitelPredicaat.soort;naam.adellijkeTitelPredicaat.soort
+naam.geslachtsnaam;naam.geslachtsnaam
+naam.volledigeNaam;naam.volledigeNaam
+volledigeNaam;naam.volledigeNaam
+naam.voorletters;naam.voorletters
+naam.voornamen;naam.voornamen
+naam.voorvoegsel;naam.voorvoegsel
+naam.aanduidingNaamgebruik;naam.aanduidingNaamgebruik
+aanduidingNaamgebruik;naam.aanduidingNaamgebruik
+naam.aanduidingNaamgebruik.code;naam.aanduidingNaamgebruik.code
+aanduidingNaamgebruik.code;naam.aanduidingNaamgebruik.code
+naam.aanduidingNaamgebruik.omschrijving;naam.aanduidingNaamgebruik.omschrijving
+aanduidingNaamgebruik.omschrijving;naam.aanduidingNaamgebruik.omschrijving
+naam.aanhef;naam.aanhef
+aanhef;naam.aanhef
+naam.aanschrijfwijze;naam.aanschrijfwijze
+aanschrijfwijze;naam.aanschrijfwijze
+naam.aanschrijfwijze.aanspreekvorm;naam.aanschrijfwijze.aanspreekvorm
+aanschrijfwijze.aanspreekvorm;naam.aanschrijfwijze.aanspreekvorm
+aanspreekvorm;naam.aanschrijfwijze.aanspreekvorm
+naam.aanschrijfwijze.naam;naam.aanschrijfwijze.naam
+aanschrijfwijze.naam;naam.aanschrijfwijze.naam
+naam.gebruikInLopendeTekst;naam.gebruikInLopendeTekst
+gebruikInLopendeTekst;naam.gebruikInLopendeTekst
+naam.inOnderzoek;naam.inOnderzoek
+naam.inOnderzoek.datumIngangOnderzoek;naam.inOnderzoek.datumIngangOnderzoek
+naam.inOnderzoek.datumIngangOnderzoek.type;naam.inOnderzoek.datumIngangOnderzoek.type
+naam.inOnderzoek.datumIngangOnderzoek.datum;naam.inOnderzoek.datumIngangOnderzoek.datum
+naam.inOnderzoek.datumIngangOnderzoek.onbekend;naam.inOnderzoek.datumIngangOnderzoek.onbekend
+naam.inOnderzoek.datumIngangOnderzoek.jaar;naam.inOnderzoek.datumIngangOnderzoek.jaar
+naam.inOnderzoek.datumIngangOnderzoek.maand;naam.inOnderzoek.datumIngangOnderzoek.maand
+naam.inOnderzoek.adellijkeTitelPredicaat;naam.inOnderzoek.adellijkeTitelPredicaat
+naam.inOnderzoek.geslachtsnaam;naam.inOnderzoek.geslachtsnaam
+naam.inOnderzoek.voorletters;naam.inOnderzoek.voorletters
+naam.inOnderzoek.voornamen;naam.inOnderzoek.voornamen
+naam.inOnderzoek.voorvoegsel;naam.inOnderzoek.voorvoegsel
+naam.inOnderzoek.aanduidingNaamgebruik;naam.inOnderzoek.aanduidingNaamgebruik
+inOnderzoek.aanduidingNaamgebruik;naam.inOnderzoek.aanduidingNaamgebruik
```

Comment:
is dit gewenst? op root niveau bestaat er namelijk een inOnderzoek property

Reply:
pfff, dat is een goeie. stel je voor om deze te verbieden?

Reply:
in overleg besloten om het nu zo te laten. als het niet op het lijstje staat komt een foutmelding, als het wel op het lijstje staat zo mappen als beschreven in de tweede kolom